duckdb_test.go

```diff
@@ -84,6 +85,61 @@ func TestConnPool(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestConnInit(t *testing.T) {
+	connector, err := NewConnector("", func(execer driver.Execer) error {
+		bootQueries := []string{
+			"INSTALL 'json'",
+			"LOAD 'json'",
+		}
+
+		for _, qry := range bootQueries {
+			_, err := execer.Exec(qry, nil)
+			if err != nil {
+				return err
+			}
+		}
+		return nil
+	})
+	require.NoError(t, err)
+	db := sql.OpenDB(connector)
+	db.SetMaxOpenConns(2) // set connection pool size greater than 1
+	defer db.Close()
+
+	// Get two separate connections and check they're consistent
+	conn1, err := db.Conn(context.Background())
```

Comment:
Maybe it makes sense to check for an error with `require.NoError(t, err)`

Reply:
done